src/Microsoft.Extensions.Configuration.AzureAppConfiguration/RequestTracingOptions.cs

```diff
@@ -51,5 +51,15 @@ internal class RequestTracingOptions
         /// Flag to indicate whether Microsoft.AspNetCore.SignalR assembly is present in the application.
         /// </summary>
         public bool IsSignalRUsed { get; set; } = false;
+
+        /// <summary>
+        /// Flag to indicate whether replica discovery is enabled.
+        /// </summary>
+        public bool IsReplicaDiscoveryEnabled { get; set; } = false;
```

Comment:
It's quite easy to know the count of replica discovered by the srv dns lookup, how about set `DiscoveredReplicaCount=x` rather than using a boolean to indicate enabled or not.